### PR TITLE
Add support for created and updated at timestamps

### PR DESCRIPTION
**Pull request by thetutlage**
After this PR all DML entities will implicitly have `created_at` and `updated_at` timestamps. There is no way to opt out of it.

### CODE REVIEW

**Review by adrien2p — APPROVED** [review]

File: packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts [hunk 1]
```diff
@@ -92,6 +125,11 @@ export function createMikrORMEntity() {
     MikroORMEntity: EntityConstructor<any>,
     field: PropertyMetadata
   ) {
+    if (SPECIAL_PROPERTIES[field.fieldName]) {
+      SPECIAL_PROPERTIES[field.fieldName](MikroORMEntity, field)
+      return
+    }
```
Comment (olivermrbl) [inline packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts:131]: thought: Should we either log or throw to inform the end user of this case?

**Review by olivermrbl — APPROVED** [review]

File: packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts (hunk 1 above)
Comment (thetutlage) [reply packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts:131]: Naah, this is when we are generating Mikro ORM entities from DML. I think the exception should be when DML receives it as part of the schema.

Let me add it there.

Comment (adrien2p) [reply packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts:131]: I dont think we have to log it as it will part of the documentation

Comment (thetutlage) [reply packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts:131]: @adrien2p Throwing exception?

Comment (adrien2p) [reply packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts:131]: I believe we can throw an exception only if the user provides it as well, as you mention

**Comment by joekendal** [comment]
This doesn't seem to work when updating objects afterwards

**Pull request merged**